lib/hammer_cli/ca_cert_manager.rb

```diff
@@ -0,0 +1,71 @@
+require 'fileutils'
+
+module HammerCLI
+  class CACertManager
+    attr_reader :ca_path
+
+    def initialize(ca_path)
+      @ca_path = ca_path
+    end
+
+    def add_ca_cert(uri)
+      raw_cert = get_ca_cert(uri)
+      cert_file = cert_file_name(uri)
+      store_ca_cert(raw_cert, cert_file)
+    end
+
+    def get_ca_cert(uri)
```

Comment:
I tried to extract this code to separate calss but I ended up either with SSLSocket wrapper which seems pointless or class more difficult to use. I'm open to suggestions otherwise I'd keep it as is.